cmd/vc-rest/startcmd/start.go

```diff
@@ -30,31 +30,44 @@ import (
 )
 
 const (
-	hostURLFlagName          = "host-url"
-	hostURLFlagShorthand     = "u"
-	hostURLFlagUsage         = "URL to run the vc-rest instance on. Format: HostName:Port."
-	hostURLEnvKey            = "VC_REST_HOST_URL"
-	edvURLFlagName           = "edv-url"
-	edvURLFlagShorthand      = "e"
-	edvURLFlagUsage          = "URL EDV instance is running on. Format: HostName:Port."
-	edvURLEnvKey             = "EDV_REST_HOST_URL"
-	blocDomainFlagName       = "bloc-domain"
-	blocDomainFlagUsage      = "Bloc domain"
-	blocDomainEnvKey         = "BLOC_DOMAIN"
-	hostURLExternalFlagName  = "host-url-external"
-	hostURLExternalEnvKey    = "VC_REST_HOST_URL_EXTERNAL"
-	hostURLExternalFlagUsage = "Host External Name:Port This is the URL for the host server as seen externally." +
+	hostURLFlagName      = "host-url"
+	hostURLFlagShorthand = "u"
+	hostURLFlagUsage     = "URL to run the vc-rest instance on. Format: HostName:Port."
+	hostURLEnvKey        = "VC_REST_HOST_URL"
+
+	edvURLFlagName      = "edv-url"
+	edvURLFlagShorthand = "e"
+	edvURLFlagUsage     = "URL EDV instance is running on. Format: HostName:Port."
+	edvURLEnvKey        = "EDV_REST_HOST_URL"
+
+	blocDomainFlagName      = "bloc-domain"
+	blocDomainFlagShorthand = "b"
+	blocDomainFlagUsage     = "Bloc domain"
+	blocDomainEnvKey        = "BLOC_DOMAIN"
+
+	hostURLExternalFlagName      = "host-url-external"
+	hostURLExternalFlagShorthand = "x"
+	hostURLExternalEnvKey        = "VC_REST_HOST_URL_EXTERNAL"
+	hostURLExternalFlagUsage     = "Host External Name:Port This is the URL for the host server as seen externally." +
 		" If not provided, then the host url will be used here." +
 		" Alternatively, this can be set with the following environment variable: " + hostURLExternalEnvKey
-	universalResolverURLFlagName  = "universal-resolver-url"
-	universalResolverURLFlagUsage = "Universal Resolver instance is running on. Format: HostName:Port."
-	universalResolverURLEnvKey    = "UNIVERSAL_RESOLVER_HOST_URL"
-	modeFlagName                  = "mode"
-	modeFlagShorthand             = "m"
-	modeFlagUsage                 = "Mode in which the vc-rest service will run. Possible values: " +
+
+	universalResolverURLFlagName      = "universal-resolver-url"
+	universalResolverURLFlagShorthand = "r"
+	universalResolverURLFlagUsage     = "Universal Resolver instance is running on. Format: HostName:Port."
+	universalResolverURLEnvKey        = "UNIVERSAL_RESOLVER_HOST_URL"
+
+	modeFlagName      = "mode"
+	modeFlagShorthand = "m"
+	modeFlagUsage     = "Mode in which the vc-rest service will run. Possible values: " +
 		"['issuer', 'verifier'] (default: issuer)."
 	modeEnvKey = "VC_REST_MODE"
 
+	localStorePathFlagName      = "local-store-path"
```

Comment:
what this flag mean?

Reply:
VCs have IDs and EDV encrypted documents have IDs. When someone calls a REST endpoint in edge-service, they will provide the VC ID, however this will not be the same as the ID of the associated encrypted document that contains the encrypted VC. So I need to map from VC ID -> EDV encrypted document ID, and I need to store that somewhere. So I store that in a local LevelDB database. This flag specifies where the database is created in the file system.

(It's like the [db-path flag in Aries-Framework-Go)](https://github.com/hyperledger/aries-framework-go/blob/6f509cae073a587e086972aaa93b9a878f88dfc2/cmd/aries-agent-rest/startcmd/start.go#L50)

Reply:
no need to use LOCAL_STORE_PATH follow credential store way